app/models/orders_package.rb

```diff
@@ -26,6 +26,7 @@ module States
   scope :get_dispatched_records_with_order_id, ->(order_id) { where(order_id: order_id, state: 'dispatched') }
   scope :designated, ->{ where(state: 'designated') }
   scope :dispatched, ->{ where(state: 'dispatched') }
+  scope :get_orders_package_by, ->(order_id) { joins(:order).where(orders: {id: order_id}) }
```

Comment:
the name `get_orders_package_by` is confusing, since we don't know "by what" unless we look at the implementation.

Suggestion `scope :for_order`